### ISSUE TITLE
Cannot import comics

### ISSUE THREAD

**Issue by midniteslice** [hmhrex/Tenma#24]
I have tried on both Ubuntu 16.04 and Debian 8.  On both machines the installation goes fine and nothing seems to be wrong but when I try to import comics nothing happens. I have tried both with and without the Comicvine api key. Below is the only messages that I get...

```
bj@bj-ThinkPad-S1-Yoga:~/venv$ source /home/bj/venv/bin/activate
(venv) bj@bj-ThinkPad-S1-Yoga:~/venv$ python manage.py runserver 10.2.2.117:8000
Performing system checks...

System check identified no issues (0 silenced).
January 29, 2017 - 11:18:41
Django version 1.10.1, using settings 'tenma.settings'
Starting development server at http://10.2.2.117:8000/
Quit the server with CONTROL-C.
[29/Jan/2017 11:18:49] "GET / HTTP/1.1" 200 2269
[29/Jan/2017 11:18:50] "GET /static/comics/style.css HTTP/1.1" 304 0
[29/Jan/2017 11:18:50] "GET /static/comics/script.js HTTP/1.1" 304 0
[29/Jan/2017 11:18:50] "GET /static/comics/bootstrap/bootstrap.min.js HTTP/1.1" 304 0
[29/Jan/2017 11:18:50] "GET /static/comics/flexslider/jquery.flexslider-min.js HTTP/1.1" 304 0
[29/Jan/2017 11:18:50] "GET /static/comics/flexslider/flexslider.css HTTP/1.1" 304 0
[29/Jan/2017 11:18:52] "GET /static/comics/tenma-logo.png HTTP/1.1" 304 0
[29/Jan/2017 11:18:52] "GET /static/comics/comicvine_logo.png HTTP/1.1" 304 0
[29/Jan/2017 11:18:56] "GET /server-settings HTTP/1.1" 200 1418
[29/Jan/2017 11:18:58] "GET /importer HTTP/1.1" 302 0
[29/Jan/2017 11:18:58] "GET / HTTP/1.1" 200 2269
[29/Jan/2017 11:19:51] "GET / HTTP/1.1" 200 2269
[29/Jan/2017 11:19:51] "GET /static/comics/flexslider/flexslider.css HTTP/1.1" 304 0
[29/Jan/2017 11:19:51] "GET /static/comics/style.css HTTP/1.1" 304 0
[29/Jan/2017 11:19:51] "GET /static/comics/bootstrap/bootstrap.min.js HTTP/1.1" 304 0
[29/Jan/2017 11:19:51] "GET /static/comics/flexslider/jquery.flexslider-min.js HTTP/1.1" 304 0
[29/Jan/2017 11:19:51] "GET /static/comics/script.js HTTP/1.1" 304 0
[29/Jan/2017 11:19:51] "GET /static/comics/tenma-logo.png HTTP/1.1" 304 0
[29/Jan/2017 11:19:51] "GET /static/comics/comicvine_logo.png HTTP/1.1" 304 0


```
This is my file structure, I have verified that the comics are not DRM'd. I have also tried it first with following the installation instructions exactly, renaming the unzipped archive to tenmaserver and copying it to the venv directory, and also as you see below, extracting directly to the venv directory

```
bj@bj-ThinkPad-S1-Yoga:~/venv/files$ tree
.
├── 3-devils-2016
│   ├── 3 Devils 1.cbz
│   ├── 3 Devils 2.cbz
│   ├── 3 Devils 3.cbz
│   └── 3 Devils 4.cbz
└── Hellblazer (1988)
    └── Hellblazer 300.cbz

2 directories, 5 files

```

**Comment by hmhrex** [comment]
This looks like it might actually be fine. Clicking the import button fires off a script that starts the import the runs in the background. If you open a new tab with `10.2.2.117:8000`, does it show any comics? I know that isn't all the intuitive to show that it's running in the background. This is something that I'm working on for the next update.

**Comment by Nerdydude101** [comment]
midniteslice, did you ever get this working for your server?  I'm having the same issue.  After reading that it might just take some time I left it for almost 16 hours and still no changes.  I've also tried both with and without the comicvine api key. My terminal messages are slightly different, but not by much: 

`(venv) miniserver@miniserver:/mnt/usb-NORELSYS_106X_0123456789ABCDE-0:0-part2/temna server/tenmaserver$ python manage.py runserver 192.168.0.120:8000
Performing system checks...

System check identified no issues (0 silenced).
March 04, 2017 - 13:22:18
Django version 1.10.1, using settings 'tenma.settings'
Starting development server at http://192.168.0.120:8000/
Quit the server with CONTROL-C.
[04/Mar/2017 13:22:24] "GET / HTTP/1.1" 200 2269
[04/Mar/2017 13:22:26] "GET /server-settings HTTP/1.1" 200 1467
[04/Mar/2017 13:22:29] "GET /importer HTTP/1.1" 302 0
[04/Mar/2017 13:22:29] "GET / HTTP/1.1" 200 2269`

Any advice from anyone?

**Comment by midniteslice** [comment]
I have not gotten it working yet. I haven't spent a whole whole lot of time
lately though. Life got in the way a bit.

**Comment by hmhrex** [comment]
Sorry that you guys are having issues getting it working!

Just to get clarification of the issue, are you just not seeing anything on the main Tenma page after running the import?

@Nerdydude101 - What OS are you running Tenma on? I'll try and test it in a VM this week to see what the issue is.

@midniteslice - I'll try and test in Ubuntu 16.04 as well this week to see what's going on there. Which flavor of Ubuntu are you running?

**Comment by midniteslice** [comment]
That's pretty much my issue. Nothing after import. 
I run debian 8 for the most part. Tested on Ubuntu 16.04 on digital ocean and on a laptop. 
Same result on both. I can spin up fedora or centos if it'll help.

⁣Sent from BlueMail ​

**Comment by russc914** [comment]
Hi,

I am having the same issue,, were you able to find a resolution to this? I am running it on Windows 10 and the output looks the same.

(venv) E:\venv\tenmaserver>python manage.py runserver 192.168.1.21:8300
Performing system checks...

System check identified no issues (0 silenced).
June 15, 2017 - 15:47:10
Django version 1.10.1, using settings 'tenma.settings'
Starting development server at http://192.168.1.21:8300/
Quit the server with CTRL-BREAK.
[15/Jun/2017 15:47:16] "GET / HTTP/1.1" 200 2269
[15/Jun/2017 15:47:16] "GET /static/comics/tenma-favicon.png HTTP/1.1" 200 5963
[15/Jun/2017 15:47:24] "GET /server-settings HTTP/1.1" 200 1467
[15/Jun/2017 15:47:27] "GET /importer HTTP/1.1" 302 0
[15/Jun/2017 15:47:27] "GET / HTTP/1.1" 200 2269
[15/Jun/2017 15:52:41] "GET / HTTP/1.1" 200 2269
[15/Jun/2017 15:52:41] "GET /static/comics/tenma-favicon.png HTTP/1.1" 200 5963
[15/Jun/2017 15:52:42] "GET / HTTP/1.1" 200 2269
[15/Jun/2017 15:52:42] "GET /static/comics/tenma-favicon.png HTTP/1.1" 200 5963
[15/Jun/2017 15:52:45] "GET /server-settings HTTP/1.1" 200 1467
[15/Jun/2017 15:52:52] "GET /importer HTTP/1.1" 302 0
[15/Jun/2017 15:52:52] "GET / HTTP/1.1" 200 2269

I do not see any indication of a background job running and I only added a couple of comics to test.
Let me me know if there is any troubleshooting I can do..

Regards,
Russ

**Comment by hmhrex** [comment]
@russc914 - I'll test with Windows 10 very shortly to try and figure out what the issue is. I haven't been able to recreate this issue so far. I'll try on a couple more machines to see if I can figure out what's going on.

**Comment by hmhrex** [comment]
I figured out the issue. This is actually because I'm running Celery as a task manager to import the comics. I forgot to add the instructions on how to get that running. It does complicate things a bit, so I'm actually going to start getting a docker image created so this process will be much much simpler.

For the time being though, I've updated the instructions in the wiki to show to get Celery running.

Windows: https://github.com/hmhrex/Tenma/wiki/Installation-on-Windows
Mac/Linux: https://github.com/hmhrex/Tenma/wiki/Installation-on-Mac-and-Linux
